src/software/ai/hl/stp/play/example_play.h

```diff
@@ -8,7 +9,7 @@
 class ExamplePlay : public Play
 {
    public:
-    ExamplePlay() = default;
+    explicit ExamplePlay(std::shared_ptr<const PlayConfig> config);
```

Comment:
from what I can tell, this is the only play that uses the explicit keyword in the constructor. Why is it important here?

Reply:
`explicit` prevents implicit casting, e.g. 
```
void foo(ExamplePlay play){} 
//omitted
std::shared_ptr<const PlayConfig> config;
foo(config); // this won't work
```

Reply:
I was more wondering why only ExamplePlay needs this explicit keyword